internal/core/command/init.go

```diff
@@ -34,14 +34,11 @@ import (
 // Global variables
 var Configuration = &ConfigurationStruct{}
 
-// Global ErrorConcept variables
-var httpErrorHandler errorconcept.ErrorHandler
-
 // BootstrapHandler fulfills the BootstrapHandler contract and performs initialization needed by the command service.
 func BootstrapHandler(wg *sync.WaitGroup, ctx context.Context, startupTimer startup.Timer, dic *di.Container) bool {
 	loggingClient := bootstrapContainer.LoggingClientFrom(dic.Get)
 	registryClient := bootstrapContainer.RegistryFrom(dic.Get)
-	httpErrorHandler = errorconcept.NewErrorHandler(loggingClient)
+	errorconcept.NewErrorHandler(loggingClient)
```

Comment:
Is this call actually needed?

Reply:
Not really, but without it we remove the logging client pull which is then going to get put right back by #2012.

Truthfully, I think we should close this PR without merge, close the issue, and handle this particular variable as a part of #2012. Anything reasonable done here is going to be immediately undone by #2012, and anything silly (like the current status of the PR) probably isn't worth doing.